app/controllers/shelters_controller.rb

```diff
@@ -1,5 +1,46 @@
 class SheltersController < ApplicationController
   def index
-    @shelters = ["Shelter1", "Shelter3", "Shelter3"]
+    @shelters = Shelter.all
```

Comment:
Storing all of the Shelter objects to an instance variable that gets passed to the view.